scripts/build/setup/termux_setup_rust.sh

```diff
@@ -27,6 +27,7 @@ termux_setup_rust() {
 	local ENV_NAME=CARGO_TARGET_${CARGO_TARGET_NAME^^}_LINKER
 	ENV_NAME=${ENV_NAME//-/_}
 	export $ENV_NAME=$CC
+	export TARGET_CFLAGS="$CFLAGS $CPPFLAGS"
```

Comment:
This Rust package pulls in the `libgit2-sys` C package, which depends on zlib.h from `TERMUX_PREFIX`, so I had to add `CPPFLAGS` to get it to compile. Of course, this will affect _all_ C packages built by cargo, so I can move this flag to the `nushell` build script alone if that might cause problems. I don't think it will, as other C packages built by cargo will likely need this include also, but it's possible.